src/store/index.js

```diff
@@ -71,7 +73,7 @@ export const store = new Vuex.Store({
                             hrsSlept: this.state.needs[0].status,
                             timeStamp: date+' '+time,
                             day: date,
-                            weekday: today.toLocaleString('en-us', {  weekday: 'long' })
+                            weekday: weekDay
                         })
                     } 
                 } 
```

Comment:
You can remove else block above (line 69), since the logic is
if there is snapshot -> update it, else add an entry (don't think it matters if db is empty or not)

Reply:
Commented out for now, want to test what happens if database is empty